### PR TITLE
1.43.2

### PR DESCRIPTION
**Pull request by denobot**
Bumped version for 1.43.2

Please ensure:
- [ ] Version in README.md is updated correctly
- [ ] Dockerfiles are updated correctly

To make edits to this PR:
```shell
git fetch upstream release_1_43.2 && git checkout -b release_1_43.2 upstream/release_1_43.2
```

cc @nathanwhit

### CODE REVIEW

File: ubuntu.dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ RUN curl -fsSL https://github.com/krallin/tini/releases/download/v${TINI_VERSION
   && chmod +x /tini
 
 
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```
Comment (mmastrac) [inline ubuntu.dockerfile:18]: Is this still necessary? I thought the sysroot build was building Ubuntu 18+ binaries.

**Review by mmastrac — APPROVED** [review]
LGTM w/Ubuntu question

File: ubuntu.dockerfile (hunk 1 above)
Comment (bartlomieju) [reply ubuntu.dockerfile:18]: I got a problem with glibc requiring 3.25 which was not available in 20.04.

Comment (bartlomieju) [reply ubuntu.dockerfile:18]: See https://github.com/denoland/deno_docker/actions/runs/9011082207/job/24758196115

**Pull request merged**